modules/ROOT/pages/org-credentials-config-mule4.adoc

```diff
@@ -3,22 +3,23 @@ ifndef::env-site,env-github[]
-Credentials are composed by two keys: Client ID and Client Secret.
+Organization credentials provide a way to uniquely identify an organization, business group, or specific environment, when linking the Mule runtime to an Anypoint organization. Credentials consists of two keys: Client ID and Client Secret.Mule runtime uses these credentials to connect and access your Anypoint organization.
 
-Mule runtime uses these credentials to connect and access your Anypoint organization.
+The recommended way to link a Mule runtime to API Manager, is by using the environment Client ID and Secret. You may also use the organization Client ID and Secret, or even business group credentials if you want to track APIs defined in Business groups while the Mule runtime is linked to a parent Business group.
 
-The recommended way to link a Mule runtime to API Manager, is by using the environment's Client ID and Secret. You may also use the organization's Client ID and Secret, or even you can use business group credentials if you want to track APIs defined in Business groups while the Mule runtime is linked to a parent Business group.
+When you use environment credentials, you will be able to manage APIs that are specific to only that environment. If your APIs are in different environments of Anypoint Platform, you must specify the organization credentials of the organization that contains both environments.
 
-The identification of a unique resource with two keys (i.e. Client ID and Client Secret) is also used when identifying a client application. +
-Client application credentials are not to be used as organization credentials. No harm is done if you choose to do so, but it will not link the Mule runtime to the organization.
+You can also use parent organization credentials to manage child organization APIs. For example, if ACME is the master organization and Logistics is a business group that lives within ACME, your API Administrator will be able to track APIs created on Logistics business group using the credentials of both Logistics and of ACME. The is because ACME is the parent organization and Logistics is the child organization.
 
-To pair your Mule runtime to your Anypoint Organization, the Runtime must start with Anypoint Platform credentials already configured.
+The identification of a unique resource with two keys, Client ID and Client Secret, is also used when identifying a client application. Client application credentials are not to be used as organization credentials. No harm is done if you choose to do so, but it will not link the Mule runtime to the organization.
+
+To pair your Mule runtime to your Anypoint Organization, the Runtime must start with Anypoint Platform credentials already configured. If you provision the adapter with organization credentials, you will be able to manage APIS on the specific adapter of that organization. 
 
 == Obtaining Credentials
 
 === Obtaining Environment's Credentials (Recommended)
```

Comment:
```suggestion
=== Obtaining Environment Credentials
```
Follow with a text transition that reiterates that this is the recommended approach, and why.